index.html

```diff
@@ -9,8 +9,21 @@
 </head>
 
 <body>
-    <header>
-        Header
+    <header id="headerHome">
+        <div id="headerImages">
+            <img src="img/Subtract.png" alt="img azul">
+            <img src="img/Sports.png" alt="img sports">
+        </div>
+        <nav id="navBar">
+            <ul id="navBarUl">
+                <li><a class="navLink" href="#">Sports</a></li>
+                <li><a class="navLink" href="#">Products</a></li>
+                <li><a class="navLink" href="#">About Us</a></li>
+                <li><a class="navLink" href="#">Training</a></li>
+                <li><a class="navLink" href="#">Community</a></li>
+            </ul>          
+        </nav>
+        <button class="lightPurpleButton">Contact</button>
```

Comment:
Please delete this class to this button.

**As it is**
![image](https://github.com/nikollquinteroch/FrontEnd-Final-Evaluation/assets/136136120/50574f0a-b13b-437f-89bf-51465d9c6cf3)

**As it should be**
![image](https://github.com/nikollquinteroch/FrontEnd-Final-Evaluation/assets/136136120/3280ed8c-a82f-42df-93b1-291536475261)

Reply:
ok